src/NServiceBus.RavenDB/SessionManagement/RavenSessionExtension.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Raven.Client;
+
+namespace NServiceBus.RavenDB.Persistence
```

Comment:
This seems like a rather hidden hidey-hole but perhaps it shouldn't be immediately available? I'm not sure what else is (or isn't) in this namespace.

Putting it in NServiceBus.RavenDB would make it more visible.